<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConvOptionsGray</class>
 <widget class="QDialog" name="WidgetConvOptionsGray">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Byte order:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonBigEndian">
       <property name="text">
        <string>Big Endian</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLittleEndian">
       <property name="text">
        <string>Little Endian</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData8">
       <property name="text">
        <string>8 bit</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData16">
       <property name="text">
        <string>16 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData32">
       <property name="text">
        <string>32 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="checkBoxMirrorBits">
       <property name="text">
        <string>Mirror bytes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxPack">
       <property name="text">
        <string>Pack</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of bits per point:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBitsPerPoint">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonBigEndian</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonLittleEndian</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData8</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData16</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData32</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBitsPerPoint</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMirrorBits</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxPack</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsGray</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePreview()</slot>
 </slots>
</ui>
